src/Components/WebAssembly/WebAssembly/src/Services/HttpClientServiceCollectionExtensions.cs

```diff
@@ -14,17 +14,18 @@ public static class HttpClientServiceCollectionExtensions
         /// configured to use the application's base address (<seealso cref="NavigationManager.BaseUri" />).
         /// </summary>
         /// <param name="serviceCollection">The <see cref="IServiceCollection" />.</param>
+        /// <param name="httpMessageHandler">Optional <see cref="HttpMessageHandler"/> to configure the </param>
         /// <returns>The configured <see cref="IServiceCollection" />.</returns>
-        public static IServiceCollection AddBaseAddressHttpClient(this IServiceCollection serviceCollection)
+        public static IServiceCollection AddBaseAddressHttpClient(this IServiceCollection serviceCollection, HttpMessageHandler httpMessageHandler = null)
```

Comment:
Now this is going beyond just setting a base address, should we rename this to `AddBrowserHttpClient` or maybe `AddWebAssemblyHttpClient`?